syntax = "proto3";

message Avion {
    string aerolinea = 1;
    string codigo_avion = 2;
    int32 peso = 3;
    int32 combustible = 4;
    string torre_control_inicial = 5;
    string torre_control_destino = 6;
}

message Destino {
    string destino = 1;
}

service Torres {
    rpc Despegar(Avion) returns (Destino) {}
}
